frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -90,7 +92,7 @@ const plugin: Plugin<ConsumedExtensions> = [
     },
   },
   {
-    type: 'ResourcePage/Detail',
+    type: 'Page/Resource/Details',
```

Comment:
nitpick: Why plural, please?

Reply:
There are several references in code to both detail and details (more plural); with no apparent consistency. It seemed to me the choice for plural made more sense as per the contents of the pages containing several sections under a tabbed navigation. This to me implied several details as oppose to a single all inclusive detail about the resource.

If there is a particular reason to use the singular form, I'm happy to go with that as I am unfamiliar with the history of this repo.

@spadgett thoughts?

Reply:
Like @christianvogt, I have a preference for "details." It's not a single detail. Resources have many properties. It's many details :)

There's no particular history that I'm aware of. We should try to be consistent.